core/embed/extmod/modtrezorutils/modtrezorutils-meminfo.h

```diff
@@ -494,13 +494,28 @@ void dump_trezor_vcp(FILE *out, const mp_obj_VCP_t *vcp) {
 }
 
 void dump_protomsg(FILE *out, const mp_obj_protomsg_t *value) {
+  mp_obj_t name[2] = {MP_OBJ_NULL, MP_OBJ_NULL};
+  mp_obj_type_t *type = protobuf_debug_msg_type();
+  type->attr((mp_obj_t)value, MP_QSTR_MESSAGE_NAME, name);
+
   print_type(out, "protomsg", NULL, value, false);
-  fprintf(out, ",\n\"shortval\": ");
-  dump_short(out, protobuf_debug_name(value));
+  fprintf(out, ",\n\"message_name\": ");
+  dump_short(out, name[0]);
```

Comment:
isn't this missing the trailing `,` in the resulting json?

Reply:
`dump_map_as_children` prints `,` as the first character